### ISSUE TITLE
Progressive Enhancements - V3

### ISSUE THREAD

**Issue by Zain-ul-din** [Zain-ul-din/whatsapp-ai-bot#74]
In the v3 release, we made this bot pretty stable. To make this more user-friendly and easy to use we can add the following progressive enhancements:

- [ ] Two videos for demo + setup guide
- [ ] Agents integration. 
- [ ] docs for Custom Model & Deployment.

feel free to suggest more features if you think any are missing.

**Comment by rikuddo91** [comment]
Is it possible to add image/document feature via bot? Like sending document/picture that ChatGPT could recognize and response to.

**Comment by Zain-ul-din** [comment]
@rikuddo91 I'm not sure about GPT however we do have the same functionality using Gemini only for image recognition.

**Comment by rikuddo91** [comment]
> @rikuddo91 I'm not sure about GPT however we do have the same functionality using Gemini only for image recognition.

Thanks. Btw, is it possible to add support for DeepSeek?

**Comment by Zain-ul-din** [comment]
@rikuddo91 yes! everything is possible go ahead and add one it would be your New Year gift for the community.
